Microsoft.Toolkit.Uwp.UI.Controls.WinRT/UniversalWindowsAppPackageFontLoader/FontCollectionLoader.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+#include "UniversalPackageFontData.h"
+
+BEGIN_NAMESPACE_CONTROLS_WINRT
+
+namespace UniversalWindowsAppPackageFontLoader
```

Comment:
Seems like a very specific namespace. This could probably go in the same namespace as the control. Does this struct need to be exposed to the developer?

Reply:
I've removed the namespace.